src/main/fc/rc_controls.h

```diff
@@ -37,12 +37,15 @@ typedef enum rc_alias {
     AUX9,
     AUX10,
     AUX11,
-    AUX12
+    AUX12,
+    COLLECTIVE = AUX3,              //woga65: for heli-like aircraft
+    GYRO_GAIN = AUX4                //...
```

Comment:
Do we need this?

It probably would be better to integrate into the existing functionality:

https://github.com/iNavFlight/inav/blob/master/docs/Inflight%20Adjustments.md

Reply:
> Do we need this?
> 
> It probably would be better to integrate into the existing functionality ...

Years ago I have tried this (inflight adjusments) approach. At first I encountered the difficulty that I could make adjustments rather in discrete steps than in absolute values. Then Michael Keller (I was still on Betaflight) told me how to use RC-Adjustment to map an RC-channel directly to the PID gain which was an undocumented feature. Sadly this did not work with INAV so I decided for myself that it is not worth the effort.

Of course one could use Inflight Adjustments to switch PID profiles according to the flight mode / headspeed, which leads to the need to have the Configurator handy on the flying field. To me this feels like a slow, cumbersome and not very flexible process.